src/components/bridge/originInfo.tsx

```diff
@@ -30,18 +31,20 @@ const OriginInfo = ({
         </Text>
         <Flex gap="1" align="center">
           <Input
-            sx={{
-              all: 'unset',
-              fontWeight: 'semibold',
-              color: 'fadedText4',
-            }}
-            w="100% !important"
-            disabled={isBalanceZero()}
+            variant="unstyled"
+            onChange={e => String(setTokenInputAmount(e.target.value))}
             placeholder="00.00"
-            type="number"
             value={tokenInputAmount}
-            onChange={e => String(setTokenInputAmount(e.target.value))}
-            autoFocus
+            color="fadedText4"
+            disabled={isBalanceZero()}
+            fontSize="lg"
+            fontWeight="semibold"
+            display="inline-block"
+            w={
```

Comment:
this formating is a bit out of the scope of the task. Make sure this does not introduce issues when people trying to add few millions in the inputs, because i recall some changes related to this because the input was too small that it was very hard to see all the digits

Reply:
I don't think it will cause any issues because its the same thing we are using for the stake page. But I will revert it now.

Reply:
@kesar, I remember I had to rework it because the UI was weird.

<img width="910" alt="image" src="https://user-images.githubusercontent.com/70170061/205700722-dd7b4869-6f56-46a6-8308-72140e2e56d5.png">

Reply:
> I don't think it will cause any issues because its the same thing we are using for the stake page. But I will revert it now.

can you add a screenshot of how it looks now if you add 3000000 in the input? try copy/paste & manual typing bcs there could be some differences that does not expand the input

Reply:
<img width="892" alt="image" src="https://user-images.githubusercontent.com/70170061/205703292-db7ccae2-2aa6-497d-8576-d8412cbe4d73.png">